index.html

```
@@ -1,39 +1,611 @@
+        </div>
+        <span class="picker-sep">:</span>
+        <div class="picker-drum" id="picker-min">
+          <div class="picker-highlight"></div>
+          <div class="picker-inner"></div>
+        </div>
+      </div>
+
+      <!-- Label -->
+      <div class="form-row">
+        <div class="form-label">Label</div>
+        <input type="text" class="form-input" id="alarm-label-input" placeholder="Alarm name" maxlength="30">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)